src/compilers/parallel.cr

```diff
@@ -1,6 +1,6 @@
 module Mint
   class Compiler
-    protected def prefix(statement : Ast::Statement, value : String)
+    protected def prefix(_node : Ast::Parallel, statement : Ast::Statement, value : String)
```

Comment:
Is this just for overloading?

Reply:
Yup, otherwise the methods would overwrite each other due to the same signatures.

Reply:
Instead of overloading we could use different names, like `try_prefix`, `parallel_prefix` and so on - if that's better.